.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+---
+version: 2.0
+
+references:
+  stack_build: &stack_build
+    docker:
+      # https://github.com/haskell-works/stack-build/blob/master/minimal/Dockerfile
+      - image: quay.io/haskell_works/stack-build-minimal
+
+    steps:
+      - checkout
+      - run:
+          name: Digest
+          command: |
+            # Build cache digests:
+            # 1. Build digest per stack.yaml (with contents) and stack args
+            echo -- "${STACK_ARGUMENTS}" > rdigest
+            md5sum $STACK_YAML >> rdigest
+
+            # 2. Build digest for all file changes
+            git ls-files | xargs md5sum > sdigest
+
+      - restore_cache:
+          keys:
+            - 1-{{ .Branch }}-{{ checksum "rdigest" }}-{{ checksum "sdigest" }}
+            - 1-{{ .Branch }}-{{ checksum "rdigest" }}-
+            - 1-{{ .Branch }}-
+            - 1-master-
+
+      - run:
+          name: Dependencies
+          command: |
+            stack setup $STACK_ARGUMENTS
+            stack build stratosphere $STACK_ARGUMENTS --dependencies-only \
+            --test --no-run-tests --jobs=1 --flag stratosphere:-library-only
```

Comment:
Can you indent this line by two spaces since it is part of the command on the previous line?